src/commands/deploy/latest.js

```diff
@@ -325,20 +325,7 @@ export default async function main(
       createArgs
     );
 
-    if (
-      firstDeployCall instanceof WildcardNotAllowed ||
-      firstDeployCall instanceof CantSolveChallenge ||
-      firstDeployCall instanceof DomainConfigurationError ||
-      firstDeployCall instanceof DomainNotFound ||
-      firstDeployCall instanceof DomainPermissionDenied ||
-      firstDeployCall instanceof DomainsShouldShareRoot ||
-      firstDeployCall instanceof DomainValidationRunning ||
-      firstDeployCall instanceof DomainVerificationFailed ||
-      firstDeployCall instanceof SchemaValidationFailed ||
-      firstDeployCall instanceof TooManyCertificates ||
-      firstDeployCall instanceof TooManyRequests ||
-      firstDeployCall instanceof InvalidDomain
-    ) {
+    if (firstDeployCall instanceof Error) {
```

Comment:
This is touching a JS file, that was my point. Since we have no inference here, probably it's not a bad idea to keep them for now but It's fine if we remove them anyway, as you prefer